en/tutorials-and-examples/blog/part-three.rst

```diff
@@ -206,6 +206,26 @@ edit template files::
     echo $this->Form->input('lft');
     echo $this->Form->input('rght');
 
+And disable or remove any associated validators from your CategoriesTable model:
+
+    public function validationDefault(Validator $validator)
+    {
+        $validator
+            ->add('id', 'valid', ['rule' => 'numeric'])
+            ->allowEmpty('id', 'create');
+
+        // Comment or delete associated validators
+        //  $validator
+        //      ->add('lft', 'valid', ['rule' => 'numeric'])
+        //      ->requirePresence('lft', 'create')
```

Comment:
Thanks Mark. As you can tell I haven't reviewed the validation documentation thoroughly.